WootricSDK/WootricSDK/WTRApiClient.m

```diff
@@ -28,12 +28,15 @@
 #import "WTRLogger.h"
 #import <CommonCrypto/CommonHMAC.h>
 
-NSString *const WootricSamplingRule = @"Wootric Sampling Rule";
+static NSString *const WTRSamplingRule = @"Wootric Sampling Rule";
+static NSString *const WTRRegisterEventsEndpoint = @"/registered_events.json";
+static NSString *const WTREligibleEndpoint = @"/eligible.json";
+static NSString *const WTRSurveyServerURL = @"https://survey.wootric.com";
+static NSString *const WTRBaseAPIURL = @"https://api.wootric.com";
+static NSString *const WTRAPIVersion = @"api/v1";
```

Comment:
You are already using the `api` subdomain. You don't need to do the `api/`part anymore

Reply:
Looks like it has always been like this 🙃 Feel free to leave it or remove it